src/main/java/pp/app/weather/WeatherResponseDto.java

```diff
@@ -0,0 +1,21 @@
+package pp.app.weather;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class WeatherResponseDto {
+
+    private String city;
+    private BigDecimal temperature;
+    private BigDecimal temp_felt;
```

Comment:
Nie stosowałbym nazw z `_` jako rozdzieleniem - to jest już Twój obiekt domenowy więc nazywaj pola zgodnie z konwencją Javową czyli camelCasem